source/stylesheets/refills/_animate.scss

```diff
@@ -0,0 +1,87 @@
+$base-accent-color: #477dca !default;
+
+.animated {
+  -webkit-animation-duration: 1s;
+          animation-duration: 1s;
+  -webkit-animation-fill-mode: both;
+          animation-fill-mode: both;
+}
+
+.animated.infinite {
+  -webkit-animation-iteration-count: infinite;
+          animation-iteration-count: infinite;
+}
+
+.animated.alternate {
+  -webkit-animation-direction: alternate;
+          animation-direction: alternate;
+}
+
+.animated.iteration {
+  -webkit-animation-iteration-count: 2;
+          animation-iteration-count: 2;
+}
+
+//  Tweak the keyframes below. Or swap them with 
+//  another animation pattern at 
+//  https://github.com/daneden/animate.css/tree/master/source
+//
+/////////////////////////////////////////////////////////////
+
+@-webkit-keyframes zoomOut {
+  0% {
+    opacity: 1;
+  }
+
+  50% {
+    opacity: 0;
+    -webkit-transform: scale3d(.3, .3, .3);
+            transform: scale3d(.3, .3, .3);
+  }
+
+  100% {
+    opacity: 0;
+  }
+}
+
+@keyframes zoomOut {
+  0% {
+    opacity: 1;
+  }
+
+  50% {
+    opacity: 0;
+    -webkit-transform: scale3d(.3, .3, .3);
+            transform: scale3d(.3, .3, .3);
+  }
+
+  100% {
+    opacity: 0;
+  }
+}
+
+.zoomOut {
+  -webkit-animation-name: zoomOut;
+          animation-name: zoomOut;
+}
+
+/////////////////////////////////////////////////////////////
+
+.animate-target {
+  -webkit-transform-style: preserve-3d; // stop element from sticking in original position
+  border-radius: 50%;
+  border: 22px solid #E2E2E3;
```

Comment:
22 because it looked right! :smile: 
I don't think it matters. Nobody is going to use this right-looking ball anyway.